docs/user/troubleshooting.md

```diff
@@ -89,3 +89,59 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 ### Installer Fails to Create Resources
 
 The easiest way to get more debugging information from the installer is to increase the logging level. This can be done by adding `--log-level=debug` to the command line arguments. Of course, this cannot be retroactively applied, so it won't help to debug an installation that has already failed. The installation will have to be attempted again.
+
+## Generic Troubleshooting
+
+Here are some ideas if none of the [common failures](#common-failures) match your symptoms.
+For other generic troubleshooting, see [the Kubernetes documentation][kubernetes-debug].
+
+### Check for Pending or Crashing Pods
+
+This is the generic version of the [*No Worker Nodes Created*](#no-worker-nodes-created) troubleshooting procedure.
+
+```console
+$ oc --config=${INSTALL_DIR}/auth/kubeconfig get pods --all-namespaces
```

Comment:
what is this ${INSTALL_DIR} ? can we point to something that setups the kubeconfig

Reply:
> what is this `${INSTALL_DIR}`?

What you passed to `--dir` when calling `openshift-install create ...`.  That's currently implied by the name, but not documented explicitly.

> can we point to something that setups the kubeconfig

In this PR, I'm just rolling forward with the pattern that [already exists in master][1], so if you'd prefer an explicit introduction to it, it could go in via a parallel PR without conflicting with this one.

[1]: https://github.com/openshift/installer/blame/32a15a5330cb39b8256634ea4d1f98e160e7328f/docs/user/troubleshooting.md#L14